src/main/java/com/bot/commands/voice/StopCommand.java

```diff
@@ -10,9 +10,7 @@ public StopCommand() {
 		this.name = "stop";
 		this.arguments = "";
 		this.help = "Stops stream and clears the current playlist";
-		this.aliases = new String[2];
-		aliases[0] = "leave";
-		aliases[1] = "fuck off";
+		this.aliases = new String[]{"leave", "fuck off"};
```

Comment:
`fuck off` -> `fuckoff` it cant be more than 1 word?

Reply:
It's 2019, *they* can be whatever *they* want to be.